recipes/perl-yaml/1.24/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "perl-yaml" %}
+{% set version = "1.24" %}
+{% set sha256 = "b58d045a72a5e1ce714f1d42b64ccddbb055a4afac10b74972dba3b6a47dffb6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: YAML-1.24.tar.gz
+  url: https://cpan.metacpan.org/authors/id/T/TI/TINITA/YAML-1.24.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl >=5.22,<6.0
+    - perl-extutils-makemaker
+    - perl-test-yaml
+    - perl-test-more
+
+  run:
+    - perl >=5.22,<6.0
+
+test:
+  impprts:
```

Comment:
This should have been `imports:`

Reply:
no special reason for that

Reply:
thanks :)

Reply:
OK, I think the best practice is to modify the main `meta.yaml`, unless you need to keep an old version around.
Updated to 1.26 in https://github.com/bioconda/bioconda-recipes/pull/9876